api/pom.xml

```diff
@@ -66,15 +66,15 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-beans</artifactId>
-      <version>${spring.framework.version}</version>
-      <scope>compile</scope>
```

Comment:
If this artifact is just needed during tests (as mentioned in the comment https://github.com/pentaho/pentaho-platform/pull/4341/files#diff-8658bb96fff6ac848cd28d3242fce067R55 ) then scope should be test `<scope>test</scope>`.

Reply:
Nop, also needed for compilation...just checked